src/jit/utils.cpp

```diff
@@ -1489,6 +1493,27 @@ void HelperCallProperties::init()
                 noThrow = true;
                 break;
 
+            case CORINFO_HELP_DBG_IS_JUST_MY_CODE:
+            case CORINFO_HELP_BBT_FCN_ENTER:
+            case CORINFO_HELP_POLL_GC:
+            case CORINFO_HELP_MON_ENTER:
+            case CORINFO_HELP_MON_EXIT:
+            case CORINFO_HELP_MON_ENTER_STATIC:
+            case CORINFO_HELP_MON_EXIT_STATIC:
```

Comment:
Are the monitor helpers really "no throw"? Seems like "exit" could throw `SynchronizationLockException`.